examples/webgpu_xr_media_layer.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js vr - 360 stereo video</title>
```

Comment:
Please include the new example in [files.json](https://github.com/mrdoob/three.js/blob/dev/examples/files.json). Otherwise it won't appear on the homepage.